sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/AlterTableDropPartitionSuite.scala

```diff
@@ -42,6 +42,18 @@ trait AlterTableDropPartitionSuiteBase extends command.AlterTableDropPartitionSu
       checkPartitions(t) // no partitions
     }
   }
+
+  test("SPARK-33941: invalidate cache after partition dropping") {
```

Comment:
nit: perhaps it's more accurate to call this "refresh cache" instead of "invalidate cache".